src/stores/calculator.store.ts

```diff
@@ -71,5 +71,57 @@ export const useCalculatorStore = defineStore("calculator", {
     removeTechnique(index) {
       this.techniques.splice(index, 1);
     },
+    setFilters() {
+      // set filter options from technique values
+      this.setTechniques();
+      const platforms = new Set();
+      const nist = new Set();
+      const cis = new Set();
+      const platform_array = [];
+      const cis_array = [];
+      const nist_array = [];
+      // get all unique filter values for each category by adding to set objects
+      for (const t of this.techniques) {
```

Comment:
I'm not sure how this slipped past `npm run type-check` and `npm run lint`. Typescript currently can't workout what type of array `this.techniques` is. Be sure to declare the array types in your `state`.

Either:
```typescript
state: () => ({
  myJson      : json,
  techniques  : [] as Array<?>,
  activeFiltersObj: {
    nist      : [] as Array<?>,
    cis       : [] as Array<?>,
    detection : [] as Array<?>,
    os        : [] as Array<?>,
  },
...
```
Filling in `?` with the type. Or, if you don't want to clutter the value declarations:

`StoreTypes.ts`:
```typescript
interface CalculatorState {
  myJson      : ?,
  techniques  : Array<?>,
  activeFiltersObj: {
    nist      : Array<?>,
    cis       : Array<?>,
    detection : Array<?>,
    os        : Array<?>,
  }
}
```

`calculator.store.ts`:
```typescript
state: (): CalculatorState => ({
  myJson      : json,
  techniques  : [],
  activeFiltersObj: {
    nist      : [],
    cis       : [],
    detection : [],
    os        : [],
  },
...
```